<launch>
    <node name="physique_car" pkg="alan_ugv" type="physique_car" output="screen">
        <rosparam command="load" file="$(find alan_ugv)/launch/configs/GAZEBO_CAR_SUBPUB_topics.yaml" />
        <param name="/max_vel_lin" type="double" value="4.0"/>
        <param name="/max_vel_ang" type="double" value="45"/>
        <param name="/pub_freq" type="int" value="50"/>
        <param name="/environs" type="string" value="VICON"/>

        <!-- static -->
        <param name="/static_ornot" type="bool" value="false"/>

        
        <!-- circle_traj -->
        <param name="/ang_vel" type="double" value="84"/>
        <param name="/radius" type="double" value="1.44"/>
        <param name="/center_x" type="double" value="0.0"/>
        <param name="/center_y" type="double" value="0.43"/>
        

    </node>
</launch>